src/main/java/com/zufar/onlinestore/payment/service/PaymentEventCatcher.java

```diff
@@ -0,0 +1,39 @@
+package com.zufar.onlinestore.payment.service;
+
+import com.stripe.model.Event;
+import com.stripe.model.PaymentIntent;
+import com.zufar.onlinestore.payment.enums.PaymentStatus;
+import com.zufar.onlinestore.payment.exception.UnexpectedPaymentStatusException;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+import java.util.Arrays;
+import java.util.Objects;
+
+@Slf4j
+@Service
+public class PaymentEventCatcher {
```

Comment:
add java doc please
it helps to understand this class for people who have no idea about stripe

Reply:
Done